post_processing/ci_plots.jl

```diff
@@ -59,41 +60,72 @@ YLOGSCALE = Dict(
     ),
 )
 
+long_name(var) = var.attributes["long_name"]
+short_name(var) = var.attributes["short_name"]
+
+function var_info(var)
```

Comment:
This function makes assumptions:
- when `slice_lat` is available, also `slice_lon` is available
- if `slice_lat` is not available, `slice_x` and `slice_y` are both available.

Reply:
Can you also add a docstring and give an example of what this function does (in the docstring)?